src/openclaw_pipeline/unified_pipeline_enhanced.py

```
@@ -1749,7 +1749,14 @@ def main():
     parser.add_argument("--batch-size", type=int, help="批次大小（用于articles/clippings）")
     parser.add_argument("--dry-run", action="store_true", help="预览模式")
     parser.add_argument("--with-refine", action="store_true", help="在 absorb/moc 之后执行 cleanup + breakdown 批处理")
-    parser.add_argument("--pack", default=None, help="Domain pack 名称（默认: default-knowledge）")
+    parser.add_argument(
+        "--pack",
+        default=None,
+        help=(
+            f"Domain pack 名称（默认兼容 pack: {DEFAULT_PACK_NAME}；"
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)